deploy-manage/upgrade/deployment-or-cluster/upgrade-717.md

```diff
@@ -1,7 +1,501 @@
 ---
 applies_to:
   stack: ga
+  serverless: unavailable
 ---
 # Upgrade from 7.17 to {{version.stack}}
 
-Placeholder for future 7.17 upgrade path doc
\ No newline at end of file
+{{stack}} version 7.17 has a defined end of support date of 15 January 2026, as stated in the [Elastic Product & Version End of Life Policy](https://www.elastic.co/support/eol). This document provides a guided plan to upgrade from 7.17 to the latest {{version.stack}} release. It complements the official upgrade documentation by showing how the different pieces fit together in a complete upgrade exercise.
+
+This guide applies to all deployment types: ({{ech}} (ECH), {{ece}} (ECE), {{eck}} (ECK), and self-managed clusters) running {{es}} version **7.17.x**. If you are using an earlier version, [upgrade first to the latest 7.17 release](https://www.elastic.co/guide/en/elastic-stack/7.17/upgrading-elastic-stack.html) before proceeding.
+
+## Overview
+
+Upgrading from 7.17 to {{version.stack}} requires two major upgrades. Each major upgrade must be prepared and executed independently, although the planning phase can be shared.
+
+1. **7.17.x → 8.19.x**
+    This brings the cluster onto the latest supported 8.x release, which is the required intermediate version before upgrading to {{version.stack}}.
+    Before running this upgrade, all ingest components and client libraries must be upgraded to 7.17.x.
+
+2. **8.19.x → {{version.stack}}**
+    This completes the upgrade to the latest 9.x release.
+    Before running this upgrade, all ingest components and client libraries must be upgraded to 8.19.x.
+
+:::{note}
+Upgrading only to version 8.19.x is also a supported path, as it remains a maintained and fully supported release. However, we recommend completing the upgrade to the latest {{version.stack}} version to take advantage of performance ongoing improvements and new features.
```

Comment:
I think this might be a littleclearer

```suggestion
Upgrading only as far as {{stack}} 8.19.x is also a supported path, as 8.19.x remains a maintained and fully supported release. However, we recommend completing the upgrade to the latest version, {{version.stack}}, to take advantage of ongoing performance improvements and new features.
```